docs/plugindev/README.md

```diff
@@ -232,6 +232,34 @@ make inventory-init
 make inventory-plugin-add
 ```
 
+### Updating the plugin group
+
```

Comment:
I suggest putting a small introduction for this section.  Maybe something like:

"A Plugin groups define a list of plugin/version combinations that are applicable together.  They facilitate an efficient installation of these plugins.  Below are instructions on how to create and publish a plugin group."